assignment2.yaml

```diff
@@ -0,0 +1,289 @@
+        type: "integer"
+        description: "The episode number of this film."
+        example: 4
+      opening_crawl:
+        type: "string"
+        description: "The opening paragraphs at the beginning of this film."
+        example: "It is a period of civil war.\n\nRebel spaceships, striking\n\nfrom a hidden base, have won\n\ntheir first victory against\n\nthe evil Galactic Empire.\n\n\n\nDuring the battle, Rebel\n\nspies managed to steal secret\r\nplans to the Empire's\n\nultimate weapon, the DEATH\n\nSTAR, an armored space\n\nstation with enough power\n\nto destroy an entire planet.\n\n\n\nPursued by the Empire's\n\nsinister agents, Princess\n\nLeia races home aboard her\n\nstarship, custodian of the\n\nstolen plans that can save her\n\npeople and restore\n\nfreedom to the galaxy...."
+      director:
+        type: "string"
+        description: "George Lucas"
+        example: "A New Hope"
+      producer:
+        type: "string"
+        description: "The name(s) of the producer(s) of this film. Comma separated."
+        example: "Gary Kurtz, Rick McCallum"
+      release_date:
+        type: "string"
+        description: "The ISO 8601 date format of film release at original creator country."
+        example: "1977-05-25"
+      species:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of species resource URLs that are in this film."
+        example: ["http://swapi.co/api/species/1/"]
+      starships:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of starship resource URLs that are in this film."
+        example: -["http://swapi.co/api/starships/2/"]
+      vehicles:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of vehicle resource URLs that are in this film."
+        example: -["http://swapi.co/api/vehicles/4/"]
+      characters:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of people resource URLs that are in this film."
+        example: -["http://swapi.co/api/people/1/"]
+      planets:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of planet resource URLs that are in this film."
+        example: -["http://swapi.co/api/planets/1/"]
+      url:
+        type: "string"
+        description: "the hypermedia URL of this resource."
+        example: "http://swapi.co/api/films/1/"
+      created:
+        type: "string"
+        description: "the ISO 8601 date format of the time that this resource was created."
+        example: "2014-12-10T14:23:31.880000Z"
+      edited:
+        type: "string"
+        description: "the ISO 8601 date format of the time that this resource was edited."
+        example: "2014-12-12T11:24:39.858000Z"
+  Films:
+    type: "array"
+    items:
+      $ref: "#/definitions/Film"  
+  FilmSchema:
+    type: "object"
+    properties:
+      required:
```

Comment:
write this
`
      required:
        type: "array"
        items:
          type: "string"
        enum: 
          - title
          - episode_id
          - opening_crawl
          - director
          - producer
          - release_date
          - characters
          - planets
          - starships
          - vehicles
          - species
          - url
          - created
          - edited
        example:
          - title
          - episode_id
          - opening_crawl
          - director
          - producer
          - release_date
          - characters
          - planets
          - starships
          - vehicles
          - species
          - url
          - created
          - edited
`